relay-metrics/benches/aggregator.rs

```diff
@@ -0,0 +1,156 @@
+use std::collections::BTreeMap;
+use std::fmt;
+
+use actix::prelude::*;
+
+use criterion::{criterion_group, criterion_main, BatchSize, BenchmarkId, Criterion};
+
+use relay_common::{ProjectKey, UnixTimestamp};
+use relay_metrics::{Aggregator, AggregatorConfig};
+use relay_metrics::{Bucket, FlushBuckets, Metric, MetricUnit, MetricValue};
+
+#[derive(Clone, Default)]
+struct TestReceiver;
+
+impl Actor for TestReceiver {
+    type Context = Context<Self>;
+}
+
+impl Handler<FlushBuckets> for TestReceiver {
+    type Result = Result<(), Vec<Bucket>>;
+
+    fn handle(&mut self, _msg: FlushBuckets, _ctx: &mut Self::Context) -> Self::Result {
+        Ok(())
+    }
+}
+
+/// Struct representing a testcase for which insert + flush are timed.
+struct MetricInput {
```

Comment:
If I understand `lazy_static` correctly, all the metrics that are inserted end up in the same bucket, right? Because they have the same name + timestamp + tags. I wonder if we should add another dimension `num_metric_names`, so that the inserted metrics are distributed over different buckets.

Reply:
right now we already have many buckets by having many projects but I think it makes sense